src/applications/gi/components/profile/SchoolLocations.jsx

```diff
@@ -75,7 +75,7 @@ export class SchoolLocations extends React.Component {
       physicalZip,
     } = institution;
     const nameLabel = this.institutionIsBeingViewed(facilityCode) ? (
-      <h6>{name}</h6>
+      <p className="main-campus-name">{name}</p>
```

Comment:
this is too specific of a class name as this is being done in the generic `renderRow` function

If this is specific to the Main Campus row changes should either be made in `renderMainRow` or by using css class `${type}-row` which is used by the `<tr>`

```
.main-row {
    h6 {
      margin-bottom: 0em;
      margin-top: 0em;
    }
}
```

Is UI/UX ok with the switch to a `<p>` tag?